serializations.md

```diff
@@ -732,21 +753,58 @@ tree.
 
 ```
 
+###### Channel off-chain pinned block for on-chain environment
```

Comment:
Shall this section be integrated with [no-pinned-block serialization](https://github.com/aeternity/aeternity/pull/2316/files#diff-ca20b5436930d4ac9286ba2ceddf844bR40)?

Reply:
This section only defines the serialization of payload, if provided. The empty payload is already described:
[close solo](https://github.com/aeternity/protocol/pull/337/files/e51c13184aef6cb914ccf079f7b474b1e6ec3162#diff-3976273847c0bfced542c67100a7990bR591)
[slash](https://github.com/aeternity/protocol/pull/337/files/e51c13184aef6cb914ccf079f7b474b1e6ec3162#diff-3976273847c0bfced542c67100a7990bR605)
[snapshot](https://github.com/aeternity/protocol/pull/337/files/e51c13184aef6cb914ccf079f7b474b1e6ec3162#diff-3976273847c0bfced542c67100a7990bR630)
and [force progress](https://github.com/aeternity/protocol/pull/337/files/e51c13184aef6cb914ccf079f7b474b1e6ec3162#diff-3976273847c0bfced542c67100a7990bR665)